### PR TITLE
enhanced tests for overdue last task expunge

### PR DESCRIPTION
**Pull request by unterstein**
enhanced tests for #4203

Related to https://github.com/mesosphere/marathon/pull/4206 (but other PR got stuck in old CI jobs)

### CODE REVIEW

**Comment by unterstein** [comment]
retest this please

**Comment by unterstein** [comment]
retest this please

File: src/test/scala/mesosphere/marathon/core/task/jobs/ExpungeOverdueLostTasksActorTest.scala [hunk 1]
```diff
@@ -60,10 +61,10 @@ class ExpungeOverdueLostTasksActorTest extends MarathonSpec with GivenWhenThen w
     noMoreInteractions(stateOpProcessor)
   }
 
-  test("a unreachable task with more then 24 hours with no status update should be killed") {
+  test("an unreachable task with more then 24 hours with no status update should be killed") {
```
Comment (meichstedt) [inline src/test/scala/mesosphere/marathon/core/task/jobs/ExpungeOverdueLostTasksActorTest.scala]: s/then/than

File: src/test/scala/mesosphere/marathon/core/task/jobs/ExpungeOverdueLostTasksActorTest.scala [hunk 2]
```diff
@@ -60,10 +61,10 @@ class ExpungeOverdueLostTasksActorTest extends MarathonSpec with GivenWhenThen w
...
     Given("one unreachable, one running tasks")
-    val running = MarathonTestHelper.minimalRunning("/running1".toPath, since = Timestamp.apply(0))
-    val unreachable = MarathonTestHelper.minimalUnreachableTask("/running2".toPath, since = Timestamp.apply(0))
+    val running = MarathonTestHelper.minimalRunning("/running".toPath, since = Timestamp.apply(0))
```
Comment (meichstedt) [inline src/test/scala/mesosphere/marathon/core/task/jobs/ExpungeOverdueLostTasksActorTest.scala]: s/Timestamp.apply(0)/Timestamp(0)

File: src/test/scala/mesosphere/marathon/core/task/jobs/ExpungeOverdueLostTasksActorTest.scala [hunk 3]
```diff
@@ -60,10 +61,10 @@ class ExpungeOverdueLostTasksActorTest extends MarathonSpec with GivenWhenThen w
...
+    val unreachable = MarathonTestHelper.minimalUnreachableTask("/unreachable".toPath, since = Timestamp.apply(0))
```
Comment (meichstedt) [inline src/test/scala/mesosphere/marathon/core/task/jobs/ExpungeOverdueLostTasksActorTest.scala]: see above

File: src/test/scala/mesosphere/marathon/core/task/jobs/ExpungeOverdueLostTasksActorTest.scala [hunk 4]
```diff
@@ -73,6 +74,24 @@ class ExpungeOverdueLostTasksActorTest extends MarathonSpec with GivenWhenThen w
     testProbe.receiveOne(3.seconds)
 
     And("one kill call is issued")
-    verify(stateOpProcessor, once).process(any)
+    verify(stateOpProcessor, once).process(TaskStateOp.ForceExpunge(unreachable.taskId))
+    noMoreInteractions(stateOpProcessor)
+  }
+
+  test("an unreachable task with less then 24 hours with no status update should not be killed") {
```
Comment (meichstedt) [inline src/test/scala/mesosphere/marathon/core/task/jobs/ExpungeOverdueLostTasksActorTest.scala]: s/then/than

**Comment by meichstedt** [comment]
LGTM, some minor comments

**Comment by meichstedt** [comment]
Thanks!

**Pull request merged**